app/src/main/java/com/example/una/FirestoreClient.java

```diff
@@ -293,6 +293,19 @@ public void fetchDonationsFirstTime(String uid, int itemsPerPage, OnSuccessListe
                 .get().addOnSuccessListener(onSuccessListener);
     }
 
+    public void getBroadcastsFirstTime(int broadcastsPage, OnSuccessListener<QuerySnapshot> onSuccessListener) {
+        broadcasts.orderBy("time", Query.Direction.DESCENDING)
+                .limit(broadcastsPage)
```

Comment:
Do you think you can change this variable name ? I'm not sure I know what its referencing